### PR TITLE
Etram 27

### PR DESCRIPTION
**Pull request by mradagonzalez**

### CODE REVIEW

**Comment by eriveraCCC** [comment]
Please, delete all .gitkeep files

File: src/main/java/com/ixxus/etram/experttrack/model/entity/article/CustomArticleEntity.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    @Id
+    @Column(name = "id_page")
+    private Integer idPage;
+
+    @Column (name = "id_project")
+    private Integer idProject;
+
+    @Column(name = "page_name")
+    private String pageName;
+
```
Comment (eriveraCCC) [inline src/main/java/com/ixxus/etram/experttrack/model/entity/article/CustomArticleEntity.java:23]: Are theses fields necessary to be specified in the @Column? I think that every field with underscore "_" in every column is mapped with a _camelCase_ format

File: src/main/java/com/ixxus/etram/config/RestTemplateConfig.java [hunk 2]
```diff
@@ -14,11 +14,12 @@
 @Configuration
 public class RestTemplateConfig {
 
-    private static final String CONFLUENCE_USERNAME = "";
-    private static final String CONFLUENCE_TOKEN = "";
+    private static final String CONFLUENCE_USERNAME = "mrada@copyright.com";
+    private static final String CONFLUENCE_TOKEN = "7ZR04azTnT9tDNK3NTztEBF2";
```
Comment (eriveraCCC) [inline src/main/java/com/ixxus/etram/config/RestTemplateConfig.java:18]: Remove credentials :)

**Review by eriveraCCC — CHANGES_REQUESTED** [review]
Check the comments, and fix some changes needed

File: src/main/java/com/ixxus/etram/experttrack/model/entity/article/CustomArticleEntity.java (hunk 1 above)
Comment (mradagonzalez) [reply src/main/java/com/ixxus/etram/experttrack/model/entity/article/CustomArticleEntity.java:23]: While it's not needed, I think is more clear to the programmer to have the column name available. This way its easier to recognise it from the query and easier to change if it was necessary

**Pull request merged**